src/Widgets/SliderWidget/SliderWidgetComponent.tsx

```diff
@@ -15,7 +15,9 @@ import './SliderWidget.scss'
 
 provideComponent(SliderWidget, ({ widget }) => {
   const showControls = widget.get('autoplay') ? widget.get('controls') : true
-  const autoplayInterval = (widget.get('autoplayInterval') ?? 5) * 1000
+  const autoplayInterval = Math.round(
```

Comment:
* [ ] FYI renaming the variable from the attribute can make sense here since the value is something else. Maybe `intervalMs` or so?